pom.xml

```diff
@@ -249,6 +249,14 @@
           </dependency>
         </dependencies>
       </plugin>
+        <plugin>
```

Comment:
if this is not wanted / constitutes a breaking change, it can be removed along with the @Override annotations.